app/src/main/java/org/mozilla/focus/fragment/onboarding/OnBoardingFirstFragment.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.fragment.onboarding
+
+import android.content.Context
+import android.os.Bundle
+import android.transition.TransitionInflater
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
+import mozilla.telemetry.glean.private.NoExtras
+import org.mozilla.focus.GleanMetrics.Onboarding
+import org.mozilla.focus.R
+import org.mozilla.focus.ext.requireComponents
+import org.mozilla.focus.ui.theme.FocusTheme
+
+class OnBoardingFirstFragment : Fragment() {
+    private lateinit var onBoardingInteractor: OnBoardingInteractor
```

Comment:
There a slight naming inconsistency: I see both "on**B**oarding" and "on**b**oarding" being used with many more usages of the latter. 
Seeing that [on mobile we do have many more usages on "on**b**oarding"](https://searchfox.org/mozilla-mobile/search?q=onboarding&path=&case=true&regexp=false) can we stick to using this form?

Reply:
Done